CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0](https://github.com/eigerco/blockstore/compare/v0.1.1...v0.2.0) - 2024-03-25
+
+### Added
+- *(blockstore)* add IndexedDb blockstore ([#221](https://github.com/eigerco/blockstore/pull/221))
```

Comment:
Manually change all the links to `[eigerco/lumina#XXX](https://github.com/eigerco/lumina/pull/XXX)`